tokenserver-db-postgres/Cargo.toml

```
@@ -6,4 +6,16 @@ edition.workspace = true
 license.workspace = true
 
 [dependencies]
-diesel = { workspace = true, features = ["postgres", "r2d2"] }
\ No newline at end of file
+async-trait.workspace = true
+deadpool = { workspace = true }
+diesel = { workspace = true, features = ["postgres", "r2d2"] }
+diesel-async = { workspace = true, features = ["postgres"] }
+diesel_logger = { workspace = true }
```

nit: we could shorthand these, somehow we ended up w/ the verbose version in a lot of places
```suggestion
deadpool.workspace = true
diesel = { workspace = true, features = ["postgres", "r2d2"] }
diesel-async = { workspace = true, features = ["postgres"] }
diesel_logger.workspace = true
```